vars/tdr.groovy

```diff
@@ -26,23 +26,42 @@ def runGitSecrets(repo) {
 //It is important for TDR devs to know that the code they want to merge doesn't break TDR. By sending the build status for every commit (all branches) to GitHub we can ensure code that breaks TDR cannot be merged.
 
 // Call this when build starts (to let person who made changes know they are being checked) - call within first 'stage' of Jenkins pipeline actions.
-def reportStartOfBuildToGitHub(String repo) {
-  withCredentials([string(credentialsId: 'github-jenkins-api-key', variable: 'GITHUB_ACCESS_TOKEN')]) {
-    sh "curl -XPOST '${githubApiStatusUrl(repo)}' -H 'Authorization: bearer ${env.GITHUB_ACCESS_TOKEN}' --data '{\"state\":\"pending\",\"target_url\":\"${env.BUILD_URL}\",\"description\":\"Jenkins build has started\",\"context\":\"TDR Jenkins build status\"}'"
+def reportStartOfBuildToGitHub(String repo, String sha='null') {
+  if (sha != 'null') {
```

Comment:
Is this if statement necessary? The default value is `null` anyway, so there's no difference between calling `githubApiStatusUrl(repo)` and `githubApiStatusUrl(repo, sha)`.

Ditto for the two methods below.